assets/wizards/engagement/components/prompt.tsx

```diff
@@ -264,37 +284,69 @@ export default function Prompt( { inFlight, prompt, setInFlight, setPrompts }: P
 							/>
 						) }
 						{ success && <Notice noticeText={ success } isSuccess /> }
-						<Button
-							isPrimary
-							onClick={ () => savePrompt( prompt.slug, values ) }
-							disabled={ inFlight }
-						>
-							{ inFlight
-								? __( 'Saving…', 'newspack' )
-								: sprintf(
-										// Translators: Save or Update settings.
-										__( '%s prompt settings', 'newspack' ),
-										prompt.ready ? __( 'Update', 'newspack' ) : __( 'Save', 'newspack' )
-								  ) }
-						</Button>
-						<WebPreview
-							url={ getPreviewUrl( prompt ) }
-							renderButton={ ( { showPreview }: { showPreview: () => void } ) => (
-								<Button
-									disabled={ inFlight }
-									isSecondary
-									onClick={ async () => {
-										if ( isDirty ) {
-											await savePrompt( prompt.slug, values );
-										}
-										showPreview();
-									} }
-								>
-									{ __( 'Preview prompt', 'newspack' ) }
-								</Button>
-							) }
-						/>
+						<div className="newspack-buttons-card">
+							<Button
+								isPrimary
+								onClick={ () => savePrompt( prompt.slug, values ) }
+								disabled={ inFlight }
+							>
+								{ inFlight
+									? __( 'Saving…', 'newspack' )
+									: sprintf(
+											// Translators: Save or Update settings.
+											__( '%s prompt settings', 'newspack' ),
+											prompt.ready ? __( 'Update', 'newspack' ) : __( 'Save', 'newspack' )
+									  ) }
+							</Button>
+							<WebPreview
+								url={ getPreviewUrl( prompt ) }
+								renderButton={ ( { showPreview }: { showPreview: () => void } ) => (
+									<Button
+										disabled={ inFlight }
+										icon={ previewIcon }
+										isSecondary
+										onClick={ async () => {
+											if ( isDirty ) {
+												await savePrompt( prompt.slug, values );
+											}
+											showPreview();
+										} }
+									>
+										{ __( 'Preview prompt', 'newspack' ) }
+									</Button>
+								) }
+							/>
+						</div>
 					</div>
+					{ helpInfo && (
+						<div className="newspack-ras-campaign__help">
+							{ helpInfo.screenshot && <img src={ helpInfo.screenshot } alt={ prompt.title } /> }
+							{ helpInfo.description && (
+								<p>
+									<span dangerouslySetInnerHTML={ { __html: helpInfo.description } } />{ ' ' }
+									{ helpInfo.url && (
+										<ExternalLink href={ helpInfo.url }>
+											{ __( 'Learn more', 'newspack' ) }
+										</ExternalLink>
+									) }
+								</p>
+							) }
+							{ helpInfo.recommendations && (
+								<>
+									<h4 className="newspack-ras-campaign__recommendation-heading">
+										{ __( 'We recommend', 'newspack' ) }
+									</h4>
+									<ul>
+										{ helpInfo.recommendations.map( ( recommendation, index ) => (
+											<li key={ index }>
+												<span dangerouslySetInnerHTML={ { __html: recommendation } } />
```

Comment:
Do we need this to be HTML? From the JSON it looks like they are just text strings.

Reply:
The current ones are just text, but I wanted to keep it flexible because we're likely to introduce other sets of prompts that can use this same setup UI, which might have other copy.